src/main/java/io/codeka/gaia/dashboard/controller/DashboardRestController.kt

```diff
@@ -0,0 +1,43 @@
+package io.codeka.gaia.dashboard.controller
+
+import io.codeka.gaia.modules.repository.TerraformModuleRepository
+import io.codeka.gaia.stacks.bo.StackState
+import io.codeka.gaia.stacks.repository.StackRepository
+import io.codeka.gaia.teams.Team
+import io.codeka.gaia.teams.User
+import org.springframework.security.access.annotation.Secured
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/dashboard")
+@Secured
+class DashboardRestController(
+    private val moduleRepository: TerraformModuleRepository,
+    private val stackRepository: StackRepository) {
+
+    @GetMapping("/summary")
+    fun summary(user: User, team: Team?): Map<String, Long> =
+        when {
+            user.isAdmin -> {
+                mapOf(
+                    "modulesCount" to moduleRepository.count(),
+                    "runningStacksCount" to stackRepository.countStacksByState(StackState.RUNNING).toLong(),
```

Comment:
Why not having the `stackRepository` returns `Long` directly?

Reply:
Indeed :disappointed: